Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -11,6 +11,18 @@ configuration:
   name: credentials
   required: false
   type: 9
+- additionalinfo: If using OAuth, the client id of the application that was created.
```

Comment:
Let's use credentials for this.  I think we can use the same credentials we use for username/password and add a checkbox "Use OAuth" and explain that when using OAuth enter into Username the cliient_id and into password the client secret.

Reply:
When using OAuth the username and password are still required for getting the first refresh token, so I think the checkbox won't be enough.